deploy/bin/gen-ed25519-key.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const tweetnacl = require('tweetnacl');
+
+function main() {
+  console.log('Generating ed25519 keypair...');
+  let keypair = tweetnacl.sign.keyPair();
+
+  fs.writeFile('docker-worker-ed25519.pub', new Buffer.from(keypair.publicKey).toString('base64'), function(err) {});
+  return fs.writeFile('docker-worker-ed25519-cot-signing-key.key', new Buffer.from(keypair.secretKey).toString('base64'), function(err) {});
+}
+
+// can't do this b/c `TypeError: Cannot read property 'catch' of undefined`
+// main().catch(err => console.log(err.stack));
```

Comment:
this is because `main` is not an `async` function, so it doesn't return a promise.